.travis.yml

```diff
@@ -47,7 +47,12 @@ matrix:
 
 install: true
 
-script: ./gradlew --no-daemon travisCiBuild -Dvariant=$VARIANT -Dscan
+script: 
+  - ./gradlew --no-daemon --version
```

Comment:
It prints a concise info of the build environment (Gradle Version, JVM, OS). It provides info for debugging and it is fast.